docs/getting-started/getting-started-nix.md

```diff
@@ -0,0 +1,44 @@
+# Getting started with Nix
+
+`Nix` is both a package manager and a programming language, in which derivations for
+said package manager are written. While `Nix` was originally developed for `NixOS`, it
+supports any Linux distribution as well as macOS. The goal of the `Nix` project is
+to provide declarative and reproducible build environments.
+
+[Stackrox-env](https://github.com/stackrox/stackrox-env) provides a development environment,
+which includes all necessary tools to contribute to the [Stackrox project](https://github.com/stackrox/stackrox) regardless
+of whether Linux or macOS is the underlying operatating system. This approach represents an
+alternative to the imperative approaches outlined in the [Linux](getting-started-linux.md) and [macOS](getting-started-darwin.md)
+guides.
+
```

Comment:
I'd highlight the advantages here a little more.
- infrastructure-as-code: you set up it once and only an occasional `git pull` is needed later to keep in sync
- hermetic
- trivial rollbacks